repository-setup.md

```diff
@@ -0,0 +1,27 @@
+# Repository Setup
+
+## Adding your tests to Prow
+
+For triggering and running tests from github, the Kubeflow org uses Prow, K8s' continuous integration tool.
+
+Follow this [pull request](https://github.com/kubernetes/test-infra/pull/7313/files) as a guide for setting up Prow.  Your pull request should:
+- Add your repository to the list of repos queried by Tide.
+- Add your respository as a presubmit and postsubmit
+- Add your pre and post submits to the test-grid
+
+Additionally make sure that the `k8s-ci-robot` is given write access to the repo and that an `OWNERS` file is added to the repository.  The `OWNERS` file, like [this one](https://github.com/kubeflow/kubeflow/blob/master/OWNERS), will specify who can review and approve on this repo.
+
+## Setting up basic tests
```

Comment:
I think this information should go in our testing repo
https://github.com/kubeflow/testing#adding-an-e2e-test-for-a-new-repository